test/metabase/test/data/postgres.clj

```diff
@@ -21,13 +21,21 @@
    :type/UUID       "UUID"})
 
 (defn- database->connection-details [context {:keys [database-name]}]
-  (merge {:host     "localhost"
-          :port     5432
+  (merge {:host     (or (env :mb-test-db-host)
+                        "localhost")
+          :port     (or (env :mb-test-db-port)
+                        5432) 
           :timezone :America/Los_Angeles}
          (when (env :circleci)
            {:user "ubuntu"})
          (when (= context :db)
-           {:db database-name})))
+           {:db database-name})
+         (when-let [password (env :mb-test-db-pass)]
```

Comment:
I think when we talked about this I suggested names like `MB_POSTGRES_TEST_DB_PASS` or the like. Because otherwise it's not obvious something like `MB_TEST_DB_PASS` is Postgres-specific. I'm in favor of adding the new env vars for configuration purposes but the need to have something in them marking them as Postgres-specific in case we want to do the same thing for MySQL